consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/IndividualRequestConsumerTest.java

```diff
@@ -120,115 +128,85 @@ public class IndividualRequestConsumerTest extends AbstractSdtUnitTestBase {
      * Method to do any pre-test set-up.
      */
     @SuppressWarnings("unchecked")
-    @Before
+    @BeforeEach
     public void setUp() {
-        mockTransformer = EasyMock.createMock(IConsumerTransformer.class);
-        mockClient = EasyMock.createMock(ITargetAppInternalEndpointPortType.class);
+        MockitoAnnotations.openMocks(this);
+
         individualRequestConsumer = new IndRequestConsumer();
         individualRequestConsumer.setTransformer(mockTransformer);
         individualRequestConsumer.setRethrowOnFailureToConnect(true);
 
         individualRequest = this.createIndividualRequest();
         individualRequestType = this.createRequestType(individualRequest);
+    }
 
+    @Test
+    void getClient() {
+        final String targetApplicationCode = "";
+        final String serviceType = "";
+        final String webServiceEndPoint = "";
+        final long connectionTimeOut = 0L;
+        final long receiveTimeOut = 0L;
+        ITargetAppInternalEndpointPortType portType = individualRequestConsumer.getClient(targetApplicationCode,

```

Comment:
Does the change to the mocking framework mean that the nested class version of IndividualRequestConsumer defined in this class is still needed?  If not then this test should be able to test the actual implementation of getClient() rather than the overridden one.

Reply:
Tried it and got complicated. Sticking with this implementation.